float32 percent_usage
rcp/CpuTimes mode_times
rcp/CpuTimes mode_usages